Binary files mythvideo-0.16.orig/mythvideo/.videogallery.cpp.swp and mythvideo-0.16/mythvideo/.videogallery.cpp.swp differ
diff -urpN mythvideo-0.16.orig/mythvideo/videogallery.cpp mythvideo-0.16/mythvideo/videogallery.cpp
--- mythvideo-0.16.orig/mythvideo/videogallery.cpp	2004-12-08 10:14:04.513967584 -0800
+++ mythvideo-0.16/mythvideo/videogallery.cpp	2004-12-08 10:14:15.618279472 -0800
@@ -25,6 +25,7 @@ using namespace std;
 #include "videoselected.h"
 #include <mythtv/mythcontext.h>
 #include <mythtv/util.h>
+#include <cmath>
 
 const int SUB_FOLDER = -1;
 const int UP_FOLDER = -2;
